kerl

```diff
@@ -953,10 +734,10 @@ _flags() {
             CC='clang'
         fi
 
-        CFLAGS="$CFLAGS" CC="$CC" "$@"
+        CFLAGS="${CFLAGS:-}" CC="$CC" "$@"
```

Comment:
I couldn't find these being used, after my changes, but ShellCheck complains about them, so I'm assuming they might come from the consumer environment, and just set them to default "".